app/helpers/sessions_helper.rb

```diff
@@ -11,16 +11,20 @@ def remember(user)
 
   def current_user
     if (user_id = session[:user_id])
-      @current_user ||= User.find(user_id)
+      @current_user ||= User.find_by(id: user_id)
```

Comment:
Why did you change this from `User.find(user_id)`? They should have the same behaviour (unless you want to use `find_by!` which will also raise an error).

Reply:
> Why did you change this from `User.find(user_id)`? They should have the same behaviour (unless you want to use `find_by!` which will also raise an error).

At user deletion, when `User.find(user_id) `was used, I encountered "ActiveRecord::RecordNotFound" error. By changing `find` with` find_by`, the method returns `nil` when the user with that certain ID is not in DB anymore.